test/uk/gov/hmrc/helptosave/connectors/EligibilityCheckConnectorSpec.scala

```diff
@@ -55,7 +55,7 @@ class EligibilityCheckConnectorSpec extends TestSupport with GeneratorDrivenProp
     "return with the eligibility check result unchanged from ITMP" in {
       forAll { result: EligibilityCheckResult ⇒
         mockGet(connector.url(nino))(Some(HttpResponse(200, Some(Json.toJson(result))))) // scalastyle:ignore magic.number
-        Await.result(connector.isEligible(nino).value, 5.seconds) shouldBe Right(result)
+        Await.result(connector.isEligible(nino).value, 5.seconds) shouldBe Right(Some(result))
```

Comment:
looks like you're still missing a unit test for the case when DES returns a 404 to this connector

Reply:
Done